### ISSUE TITLE
CC0091 on method Foo does not replace occurences like this.Foo used in delegates

### ISSUE THREAD

**Issue by jakubsuchybio** [code-cracker/code-cracker#677]
Hi.
I created code as proof of bug:
```
		class X
		{
			public static int f = 0;

			public void Foo()
			{
				f = 5;
			}
		}

		class Y : X
		{
			public event Action Changed;

			public void Goo()
			{
				Changed += this.Foo;
			}
		}
```
When I apply correction of CC0091 onto Foo, occurrence of this.Foo; doesn't change and then this occurrence gives error, because it doesn't know this.Foo only Foo.
Regards

Edit: And I am stupid, because this issue has already been reported. Sorry

**Comment by giggio** [comment]
Bug confirmed on the code fix. It should change to:

````csharp
class X
{
    public static int f = 0;

    public static void Foo()
    {
        f = 5;
    }
}

class Y : X
{
    public event Action Changed;

    public void Goo()
    {
        Changed += Foo;
    }
}
````

It should had changed `this.Foo` to `Foo` on the `Goo` method body.

**Comment by martinzimmermann** [comment]
I would like to fix this

**Comment by giggio** [comment]
@martinzimmermann go for it.

**Comment by giggio** [comment]
Closed on #691.
